### PR TITLE
[Feature] Will add Lazy Loading of Images

### PR DESCRIPTION
**Pull request by Thraxis**
This is a Proof of Concept for using Lazy Loading of images

+ Adds jsquery.unveil.js to handle lazy loading
+ Lazy Loading enabled for main show page on all layouts
+ Lazy Loading enabled on Add Recommended pages
+ Fixed a bug on poster layout on main show page causing unnecessary page reloads.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: views/partials/home/banner.mako [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%!
+layz<%!
```
Comment (OmgImAlexis) [inline views/partials/home/banner.mako]: ?

**Comment by p0psicles** [comment]
I think you can disable it for recommended shows. As 90% of the time it's waiting on backend. So winning a few ms won't matter much.  For poster layout it can definitely be an improvement.

**Comment by Thraxis** [comment]
It will still provide a benefit to mobile users as it doesn't download all the images, if they are very far from being on screen.. saving bandwidth as only the posters they view will be downloaded.

**Comment by p0psicles** [comment]
Ooh okay. That makes sense. Tnx for explaining!

**Comment by duramato** [comment]
Just gave this a test and, I can use Poster home view once again.. :heart:

**Comment by wimpyrbx** [comment]
tested and it works very nice!

**Comment by OmgImAlexis** [comment]
@Thraxis make sure all quotes a singles and I'll be happy to approve this. @p0psicles all good?

**Comment by p0psicles** [comment]
Yeah looks good

**Pull request merged**